gap/Magic.gi

```diff
@@ -332,6 +332,7 @@ function( arg )
             fi;
             Append( gapdoc.files, AUTODOC_FindMatchingFiles(pkgdir, gapdoc.scan_dirs, [ "g", "gi", "gd" ]) );
         fi;
+        Add( gapdoc.files, Filename( doc_dir_rel, "_Chunks.xml" ) );
```

Comment:
Should we perhaps only generate this file if there actually are chunks? Otherwise, lots of projects using AutoDoc but not (yet) using chunks may have to update their `.gitignore` files.

Then again, this is a fairly minor nuisance, and working around it may not be worth the added complexity. Projects using AutoDoc could/shold simply have `/doc/_*.xml` in their `.gitignore`. I'll update the `PackageMaker` template to do that right away.

Reply:
Yeah, I will change this, it is actually no problem. The only minor thing is that we give a filename to GAPDoc to scan for stuff that might not exist. We just need to check if GAPDoc complains in that case.